docs/assets/commonMarkdown/sharedDataCNconf.md

```diff
@@ -0,0 +1,20 @@
+
+**Before starting CNs**, add the following configuration items in the CN configuration file **cn.conf**:
```

Comment:
we suggest still use *BE* in v3.0.x version and use *CN* in v3.1.x and later releases, *CN* can't be used in v3.0 release.

Reply:
OK, thanks. So we need separate docs for 3.0 and 3.1+ :(